app/assets/javascripts/discourse/widgets/post-cooked.js.es6

```diff
@@ -212,6 +212,9 @@ export default class PostCooked {
       expandContract = iconHTML(desc, { title: "post.expand_collapse" });
       $(".title", $aside).css("cursor", "pointer");
     }
+    if(this.attrs.topic.ignored_usernames.includes($($aside).find('.title').text().trim().slice(0, -1))) {
+      $($aside).find("p").replaceWith("<i>Hidden content</i>");
```

Comment:
- This doesn't seem not to be the cleanest approach
- I can use the `I18n.t()` for the content

Reply:
If your 🕷 sense tells you it's not the cleanest approach, you should definitely trust it 😉

Reply:
But since we only have the username to work with on the client-side, I can't think of a better solution that doesn't require changing the way we cook quotes (ie. by adding a `data-user-id` attribute`).

Reply:
Yes, that's what I thought as well. The way the posts are cooked doesn't have a reliable identifier `data-*` to use, and therefore, I fall back to the `.title text` approach.